test/src/kbasesearchengine/test/parse/ObjectParserTest.java

```diff
@@ -0,0 +1,256 @@
+package kbasesearchengine.test.parse;
+
```

Comment:
Speaking of method sigs, this is a good time to clean them up. My rules are:
1) all inputs are `final` unless there's a really good reason not to be
2) if the signature can't entirely fit on one line then:
  a) Move the throws statement to the next line, indent an extra 8 spaces. If another line for exceptions is needed, indent an additional 4.
  b) If the sig still can't fit on one line then put each variable on its own line.

Reply:
👍

Reply:
I don't think all the sigs fit these rules yet?